package exception;

public class Throw1  {
  public static void main(String[] args) {

	//throw new ArithmeticException("erorr");
	  if(true)
	  {
	  throw new ArithmeticException();
	  }
	  else
	  {
	  System.out.println("After throw statement...!");
	  }
	  }
	  
}

